src/Admin/AdminInterface.php

```diff
@@ -726,6 +734,60 @@ public function getListMode();
      */
 //    NEXT_MAJOR: uncomment this method in 4.0
     // public function isDefaultFilter(string $name): bool;
+
+//    NEXT_MAJOR: uncomment this method in 4.0
+//    public function setFilterPersister(?\Sonata\AdminBundle\Filter\Persister\FilterPersisterInterface\FilterPersisterInterface $filterPersister = null): void;
```

Comment:
If we are speaking exacly about this error, then it looks like this
```
		-
		    # will be fixed in v4. Currently BC break
			message: "#^Call to an undefined method Sonata\\\\AdminBundle\\\\Admin\\\\AdminInterface\\:\\:setFilterPersister\\(\\)\\.$#"
			count: 1
			path: src/Action/RetrieveAutocompleteItemsAction.php

```

you can map all the new interface methods with errors in PHPStan baseline file

Reply:
Maybe we shouldn't consider that adding these methods in the interface are BC-break them. Since these methods are needed.

What do you think @sonata-project/contributors ?

Reply:
this is BC break. Someone may implement this `AdminInterface` and after update bit will fail people's cide

Reply:
But I think there should be GitHub action that runs https://github.com/Roave/BackwardCompatibilityCheck

But that's other PR.

Reply:
That would be great indeed.

Reply:
https://github.com/sonata-project/SonataAdminBundle/issues/6164

Created an issue